internal/compute/template.go

```diff
@@ -191,23 +190,16 @@ type MetaEnvironment struct {
 	Email   string
 }
 
-var builtinVariables = []string{
-	"repo",
-	"path",
-	"content",
-	"commit",
-	"author",
-	"date",
-	"email",
-}
+var empty = struct{}{}
 
-func isBuiltinVariable(str string) bool {
-	for _, v := range builtinVariables {
-		if v == str {
-			return true
-		}
-	}
-	return false
+var builtinVariables = map[string]struct{}{
+	"repo":    empty,
+	"path":    empty,
+	"content": empty,
+	"commit":  empty,
+	"author":  empty,
+	"date":    empty,
+	"email":   empty,
 }
```

Comment:
Oh, I was thkinking the values would be the template variable

```suggestion
var builtinVariables = map[string]struct{}{
	"repo":    `{{ .Repo }}`,
	"path":   `{{ .Path }}`,
	"content": `{{ .Content }}`,
	"commit":  `{{ .Commit }}`,
	"author":  `{{ .Author }}`,
	"date":   `{{ .Date }}`,
	"email":   `{{ .Email }}`,
}
```

Reply:
Ah! yeah we can do this way :-) This nudges me even more to just yoink some substitution code and write my own :P

Reply:
Cool. As always, feel free to ignore if you're not at the point this makes sense.